src/Command/AddDatasetLinkCommand.php

```diff
@@ -0,0 +1,144 @@
+            return(1);
+        }
+
+        // Accept only known types, 'NCEI' or 'ERDDAP'.
+        if (!in_array($type, array(DatasetLink::LINK_NAME_CODES["erddap"]["name"], DatasetLink::LINK_NAME_CODES["ncei"]["name"]))) {
+            $io->error("Please specify either ERDDAP or NCEI.");
+            return(1);
+        }
+
+        $dataset = $this->entityManager->getRepository(Dataset::class)->findOneBy(array('udi' => $udi));
+        if (!($dataset instanceof Dataset)) {
+            $io->error('Could not find a dataset with UDI (' . $udi . ')');
+            return(1);
+        } else {
+            // get submission
+            $submission = $dataset->getDatasetSubmission();
+            if ($submission instanceof DatasetSubmission) {
+                // create dataset link
+                if (
+                    ($type === DatasetLink::LINK_NAME_CODES["erddap"]["name"] and !empty($submission->getErddapUrl())) or
+                    ($type === DatasetLink::LINK_NAME_CODES["ncei"]["name"] and !empty($submission->getNceiUrl()))
+                ) {
+                    $io->warning("$udi already has link of type $type. Not changing.");
+                } else {
+                    $link = new DatasetLink();
+
+                    // common elements
+                    $link->setProtocol('https');
+                    $link->setCreator($systemPerson);
+                    $link->setModifier($systemPerson);
+                    $link->setCreationTimeStamp(new DateTime('now', new DateTimeZone('UTC')));
+                    $link->setName($type);
+                    $link->setUrl($targetUrl);
+
+                    if ($type === DatasetLink::LINK_NAME_CODES["erddap"]["name"]) {
+                        $linkDescription =
+                        'ERDDAP infomation table listing individual dataset files links for this dataset. '
+                        . 'Table is also available in other file formats (.csv, .htmlTable, .itx, .json, '
+                        . '.jsonlCSV1, .jsonlCSV, .jsonlKVP, .mat, .nc, .nccsv, .tsv, .xhtml) via a RESTful '
+                        . 'web service.';
+                        $link->setFunctionCode('download');
+                    } else {
+                        $linkDescription = 'Navigate directly to the URL for a descriptive web page with download links.';
+                        $link->setFunctionCode('information');
+                    }
+                    $link->setDescription($linkDescription);
+
+                    $submission->addDatasetLink($link);
+                    $this->entityManager->persist($submission);
+                    $this->entityManager->flush();
+                    $io->success("Set $type URL on $udi");
+                }
+            } else {
+                $io->error("$udi has no submission.");
+                return(1);
+            }
+        }
+        $io->success('Command Complete.');
+        return(0);
```

Comment:
Return is not a function.
```suggestion
        return 0;
```